SPOOP/Assets/Scripts/CameraController.cs

```diff
@@ -11,6 +11,7 @@ public class CameraController : MonoBehaviour {
 
 	void Start ()
 	{
+		player = GameObject.FindGameObjectWithTag ("Player");

```

Comment:
Since the player is no longer "officially" in the scene hierarchy (and is being kept between scenes) we should make player a private variable because there is nothing to attach to it.